extensions/interactions/guppy.html

```diff
@@ -1,33 +1,56 @@
-<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-175999/build/guppy-default.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
-<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-175999/build/guppy.min.js"></script>
 
 <script type="text/javascript">
   Guppy.init({
-    "path": "/third_party/static/guppy-b5055b/build"
+    'path': '/third_party/static/guppy-175999/build'
   });
+  Guppy.configure('buttons', ['controls']);
+  Guppy.configure('empty_content', '\\color{grey}{\\text{\\small{Type a formula here.}}}');
+
+  // Remove symbols since they are not supported.
+  var symbols_to_remove = ['norm', 'utf8', 'text', 'sym_name', 'eval', 'floor',
```

Comment:
When a line of code splits over multiple lines, break after the opening bracket ('[' in this case).

Reply:
Done.